pkgs/applications/misc/lutris/default.nix

```diff
@@ -96,16 +97,17 @@ in buildPythonApplication rec {
   ] ++ gstDeps;
 
   propagatedBuildInputs = [
-    evdev
+    dbus-python
     distro
+    evdev
+    importlib-metadata
```

Comment:
The repo does not contain any reference to importlib-metadata. https://github.com/lutris/lutris/search?q=importlib-metadata

Reply:
`importlib_metadata` is the module

Reply:
ah, yes, you're right, it was from the keyring dependency
```
  File "/nix/store/v8by6mqpn6gx361zlq2g97rd4mw7lwd1-python3.8-keyring-23.0.1/lib/python3.8/site-packages/keyring/backend.py", line 12, in <module>
    import importlib_metadata as metadata
ModuleNotFoundError: No module named 'importlib_metadata'
```